_includes/integrations/vue-tech-ref.md

```diff
@@ -0,0 +1,367 @@
+## TinyMCE Vue.js technical reference
+
+**Covered in this section:**
+
+- [Installing the TinyMCE Vue.js integration using NPM](#installingthetinymcevuejsintegrationusingnpm)
+- [Using the TinyMCE Vue.js integration](#usingthetinymcevuejsintegration)
+- [Configuring the editor](#configuringtheeditor)
+
+    - [`api-key`](#api-key)
+    - [`cloud-channel`](#cloud-channel)
+    - [`disabled`](#disabled)
+    - [`id`](#id)
+    - [`init`](#init)
+    - [`initial-value`](#initial-value)
+    - [`inline`](#inline)
+    - [`modelEvents`](#modelevents)
```

Comment:
`model-events`? Same in that section

Reply:
updated all instances of `modelEvents` to `model-events`